addon/templates/components/polaris-resource-list/bulk-actions.hbs

```diff
@@ -0,0 +1,139 @@
+<div
+  class="Polaris-ResourceList-BulkActions__Group
+    Polaris-ResourceList-BulkActions__Group--smallScreen
+    {{if selectMode 'Polaris-ResourceList-BulkActions__Group--entered' 'Polaris-ResourceList-BulkActions__Group--exited'}}"
```

Comment:
Nit: can you multi-line this guy please? Also the single quotes seem to be causing a template lint error 😬